exporter/opentelemetry-exporter-zipkin-json/src/opentelemetry/exporter/zipkin/json/__init__.py

```diff
@@ -145,4 +179,8 @@ def export(self, spans: Sequence[Span]) -> SpanExportResult:
         return SpanExportResult.SUCCESS
 
     def shutdown(self) -> None:
-        pass
+        if self.done:
+            logger.warning("Exporter already shutdown, ignoring call")
+            return
+        self.session.close()
```

Comment:
what does this do to in-flight requests?

Reply:
It doesn't affect the in-flight requests. Internally it uses this pool manager https://urllib3.readthedocs.io/en/1.24.3/reference/index.html#urllib3.poolmanager.PoolManager.clear.

Reply:
So it won't drop them but it won't wait for them to finish either? I think we should probably wait for them to finish if it can be done easily.